chain/jsonrpc/src/api/mod.rs

```diff
@@ -109,12 +110,12 @@ mod params {
         Result<Result<T, RpcParseError>, Value>,
     );
 
-    impl<T> Params<T> {
-        pub fn new(params: Value) -> Self {
+    impl<T> ParamsExt<T> for Params<T> {
```

Comment:
I expected these implementations to be default ones in the `ParamsExt` trait

Reply:
I left trait, but moved struct&implementation from `mod.rs` to the new crate, so now everything in one file and you also can easily add own implementation using public trait